### ISSUE TITLE
✍️ Contribution period: Ajoke Yusuf

### ISSUE THREAD

**Issue by Ajoke23** [ersilia-os/ersilia#984]
### Week 1 - Get to know the community

- [ ] Join the communication channels
- [ ] Open a GitHub issue (this one!)
- [ ] Install the Ersilia Model Hub and test the simplest model
- [ ] Install Docker if needed, and test another model
- [ ] Write a motivation statement to work at Ersilia
- [ ] Submit your first contribution to the Outreachy site

### Week 2 - Get Familiar with Machine Learning for Chemistry

- [ ] Select a model from the list suggested in GitBook
- [ ] Download and serve the model via the Ersilia Model Hub to ensure it works
- [ ] Open a repository on your GitHub user with all the necessary files
- [ ] Select and clean a dataset of 1000 molecules (example notebook 1)
- [ ] Run predictions for the molecules on the selected model and evaluate the results

### Week 3 - Validate a Model in the Wild

- [ ] Find a suitable dataset with sufficient experimental results
- [ ] Clean and standardize the dataset
- [ ] Run predictions and calculate metrics.

### Week 4 - Prepare your final application

- [ ] Submit the final application in the Outreachy website

**Comment by Ajoke23** [comment]
Week 1

**Comment by Ajoke23** [comment]
**DAY 4 (8th of March 2024)** 
**MOTIVATION STATEMENT**
I'm Ajoke Yusuf, a Data Scientist, Machine Learning enthusiast, and SDG 3 advocate. I'm a hardworking, resourceful, goal-oriented individual who possesses strong analytical and problem-solving skills with an unending quest for knowledge. I pride myself on being a fast learner and honed strong skills in problem-solving and research.
My last experience of contributing to Ersilia during the contribution stage last year in October 2023, I decided to apply again hoping Ersilia project will be there because I had a wonderful and learnable experience coupled with the amazing community. Receiving the Outreachy email and checking the project named, I decided to scroll down to alphabet "E" to look for Ersilia and I felt so excited seeing Ersilia project. 

My main aim of choosing Ersilia project is because their aim, and mission align with my goal and career objective as an impact maker and an SDG 3 advocate. One of the life experiences that ignited my interest in drug discovery due to my personal experience with cerebral malaria that almost took my life and also the death of a friend who lost her life due to sickle cell.

As an Engineering graduate living in Nigeria, the increasing mortality rate of infectious diseases in Nigeria and sub-Saharan Africa is alarming, hence I developed an interest in the biomedical field. Based on statistics, according to UNICEF (United Nations International Children Emergency Funds), **_infectious disease is the major cause of the mortality rate in children ≤ 5 years_** which was cited from this [article](https://data.unicef.org/topic/child-survival/under-five-mortality/). Research from NIH (National Library of Medicine) & NCBI (National Centre for Biotechnology Information) confirms that _**"The infrastructure and level of support for surveillance, research, and training on emerging infectious diseases in Africa are extremely limited"**_ which was cited from this [article](https://www.ncbi.nlm.nih.gov/books/NBK99567/#:~:text=At%20a%20time%20when%20increasing,%2C%20yellow%20fever%2C%20and%20trypanosomiasis).

With my knowledge in Python for Data Science, a bit knowledge in Machine Learning in conjunction with strong analytical and research skill, I believe that contributing to this project will help me garner knowledge and technical skills that will help in advancing and improving health research in Nigeria, Africa and eventually the world.

**If accepted for the 3 months internship**, I'll commit myself to bringing suggestions, undergoing research, and collaborating with the Ersilia team while learning and honing skills in Artificial Intelligence and Machine Learning. This period of internship will help propel my research and problem-solving skills which will be useful in the long run for the advancement of technology in the health sector, improving and making a sustainable impact in health research in Nigeria because I fiercely
believe that the availability & accessibility of scientific tools and data-driven insights is necessary towards solving prevalent health challenges because as a young lady living in Nigeria, an underdeveloped and low-income country, I have experienced challenges encountered in having accessibility tools for prevalent infectious diseases in my community and country (Nigeria) at large.
**After the internship**, I plan on utilizing the skills gained to improve and sustain health research tools, solve prevalent health disease issues in Nigeria, and reduce the mortality rate caused by infectious diseases. Thus, propelling sustainable research skills that will leave a long-lasting impact in the health sector in my community, Nigeria, sub-Saharan Africa, and eventually, globally.

**Comment by Ajoke23** [comment]
**WEEK 2**

**Comment by Ajoke23** [comment]
Hi @DhanshreeA I can see that you reacted to my comment here. Any feedback so far?
My week 2 task 2 (model reproducibility is still in progress) but I will appreciate if any feedback is given regarding week 2 (task 1)

**Comment by Ajoke23** [comment]
Hi @DhanshreeA I can see that you reacted to my comment here. Any feedback so far?
My week 2 task 2 (model reproducibility is still in progress) but I will appreciate if any feedback is given regarding week 2 (task 1)

**Comment by Ajoke23** [comment]
Hi @DhanshreeA I can see that you reacted to my comment here. Any feedback so far?
My week 2 task 2 (model reproducibility is still in progress) but I will appreciate if any feedback is given regarding week 2 (task 1)

**Comment by DhanshreeA** [comment]
Hi @Ajoke23, thank you for your patience, and great progress so far! Few comments:
1. I see that you are using rdkit module to standardize smiles, whereas this is only creating canonical smiles. To standardize, you have to use the standardizer module as shown through the example code in the model validations template provided by us.
2. I don't see an implementation of the model from the paper, rather only the use of the ersilia implementation. Any reason why that's the case? Please point me to it if I have missed it.

**Comment by Ajoke23** [comment]
Hi @DhanshreeA I can see that you reacted to my comment here. Any feedback so far?
My week 2 task 2 (model reproducibility is still in progress) but I will appreciate if any feedback is given regarding week 2 (task 1)

Hi @DhanshreeA Thank for feedback

**Comment by Ajoke23** [comment]
> Hi @Ajoke23, thank you for your patience, and great progress so far! Few comments:
> 1. I see that you are using rdkit module to standardize smiles, whereas this is only creating canonical smiles. To standardize, you have to use the standardizer module as shown through the example code in the model validations template provided by us.
> 2. I don't see an implementation of the model from the paper, rather only the use of the ersilia implementation. Any reason why that's the case? Please point me to it if I have missed it.

Hi @DhanshreeA. Thank you for the feedback.
1. I used the standardized python code which is found in the data cleaning notebook. But I will remove the import rdkit module

2. It's found in the reproducibility notebook.

**Comment by Ajoke23** [comment]
Hi @DhanshreeA Can I move to week 3 now since I have made the necessary changes based on your feedback?
I look forward to receiving feedback from you. Thank you for time.

**Comment by GemmaTuron** [comment]
Thansk @Ajoke23  we will provide feedback within the day today!

**Comment by Ajoke23** [comment]
> Thansk @Ajoke23  we will provide feedback within the day today!

Thank you so much for @GemmaTuron

**Comment by DhanshreeA** [comment]
Hi @Ajoke23, this looks good to me. Please proceed to task 3, and I will review it on Monday. Thereafter you can work on your final application.

**Comment by Ajoke23** [comment]
> Hi @Ajoke23, this looks good to me. Please proceed to task 3, and I will review it on Monday. Thereafter you can work on your final application.

Alright @DhanshreeA . Thanks a lot for the feedback. I will start immediately

**Comment by Ajoke23** [comment]
WEEK 3: VALIDATE MODEL IN A WILD

**Comment by DhanshreeA** [comment]
Hi @Ajoke23 any updates to report?

**Comment by Ajoke23** [comment]
> Hi @Ajoke23 any updates to report? 

Yes, there is. It will be updated shortly. I have a little challenges

**Comment by Ajoke23** [comment]
Hi @DhanshreeA @GemmaTuron I have gone ahead to start preparing for my final application. My project template is in progress and will be completed before the end of today.
I have a question, my week 3 was updated today. Can I still go ahead to create a final application without a feedback?
Will it affect my chances of getting accepted because I updated the week 3 task today.
